Change Password, PDF/Excel and Table Logic

ninaey opened the pull request:
Add export to excel file function for coordinator for Download user list report requirement

ninaey committed:
Export to excel file Function

Add export to excel file function for coordinator for Download user list report requirement

ninaey committed:
Export to pdf and color coding

Add export to excel file function for coordinator for Download user list report requirement + add back color coding for every roles that missing from before

ninaey committed:
Show Total User Count and Current View Range

Add Show Total User Count and Current View Range in the userRegister.blade.php

ninaey committed:
Login Interface

Update Login interface

SiapaJoJo reviewed (approved):
exporting excel file working functionally

SiapaJoJo reviewed (approved):
![image](https://github.com/user-attachments/assets/66aeecc0-1e87-406f-861c-f9568f97d057)

ninaey committed:
Update Manage User name

Update the page name from User Management to Manage User to match the navigation bar

ninaey committed:
Student-supervisor

Added column supervisor that assign to students

ninaey committed:
Single register

Add single registration function
